tests/e2e/specs/frontend/cart.test.js

```diff
@@ -77,15 +77,22 @@ describe( `${ block.name } Block (frontend)`, () => {
 	} );
 
 	it( 'Shows the freshest cart data when using browser navigation buttons', async () => {
-		await jest.setTimeout( 60000 );
+		//await jest.setTimeout( 60000 );
```

Comment:
Is this a mistake?

Reply:
I think it should be removed entirely, actually. Thanks for catching it. Running tests again with it removed to ensure they finish in time.